release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -59,6 +59,20 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-13-oc"]
 === OpenShift CLI (oc)
 
+[id="ocp-4-12-import-image"]
```

Comment:
Should this id be ocp-4-13-import-image?  The 4.12 in 4.13 RNs has me a bit confused.